deploy/deploy-operator-cs/MyStack.cs

```diff
@@ -0,0 +1,289 @@
+using Pulumi;
+using Kubernetes = Pulumi.Kubernetes;
+using Pulumi.Kubernetes.Types.Inputs.Core.V1;
+using Pulumi.Kubernetes.Types.Inputs.Apps.V1;
+using Pulumi.Kubernetes.Types.Inputs.Meta.V1;
+using Pulumi.Kubernetes.Types.Inputs.Rbac.V1;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        var crds = new Kubernetes.Yaml.ConfigFile("crds", new Kubernetes.Yaml.ConfigFileArgs{
+            File = "https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/v1.2.0/deploy/crds/pulumi.com_stacks.yaml"
```

Comment:
Need to remove status field with transformation?

Reply:
See above.